subsys/net/lib/conn_mgr/conn_mgr.c

```diff
@@ -191,6 +191,68 @@ K_THREAD_DEFINE(conn_mgr, CONFIG_NET_CONNECTION_MANAGER_STACK_SIZE,
 		(k_thread_entry_t)conn_mgr_handler, NULL, NULL, NULL,
 		THREAD_PRIORITY, 0, 0);
 
+/*
+ * Note to reviewers:
+ *
+ * In light of comments made on the original RFC, rather than implement these as net_mgmt requests,
+ * I have implemented them directly as net_conn_mgr functions.
+ *
+ * Does this seem appropriate? Would net_mgmt requests be better, despite the fact it would
```

Comment:
IMO the current proposal looks pretty neat.

Reply:
@rlubos  Just to be clear, do you mean the RFC issue, or what is in this PR?